### PR TITLE
[Content Fragment] Reintroduce deletion of deprecated fields in CF model

### PR DESCRIPTION
**Pull request by philipperolet**
## Description

Revert "[Content Fragment] 2 step update for migration related to #4439 (#4445)"

This reverts commit bd11c480136f20252ec701c1c3464768647ad9f4 that was only done to allow migration.

See related PR #4439 and #4445 for details

### CODE REVIEW

File: front/lib/resources/storage/models/content_fragment.ts [hunk 1]
```diff
@@ -19,15 +19,12 @@ export class ContentFragmentModel extends Model<
...
   declare contentType: ContentFragmentContentType;
   declare sourceUrl: string | null; // GCS (upload) or Slack or ...
 
-  // Fields below are set of all types that can be transduced to text or
-  // are textual in nature (PDF, text, CSV, future: .docx...)
-  declare textUrl: string | null; // url to GCS
-  declare textBytes: number | null; // number of textUrl bytes
+  // The field below should be set for all fragments that are converted to text
+  // before being put in model context (PDF, text, CSV, future: .docx...)
+  declare textBytes: number | null;
```
Comment (flvndvd) [inline front/lib/resources/storage/models/content_fragment.ts:27]: So we don't store the URL anymore?

File: front/migrations/20240322_content_fragments_refactor.ts [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    const fileUrl = await storeContentFragmentText({
+      workspaceId,
+      conversationId,
+      messageId: cfMessage.sId,
+      // @ts-expect-error content was removed from ContentFragmentResource
+      content: cf.content,
+    });
+    await cf.update({
+      // @ts-expect-error textUrl was removed from ContentFragmentResource
+      textUrl: fileUrl,
```
Comment (flvndvd) [inline front/migrations/20240322_content_fragments_refactor.ts:31]: I'm probably missing something but we remove `textUrl` from the model and in this migration we try to set it 🤔.

Comment (philipperolet) [reply front/migrations/20240322_content_fragments_refactor.ts:31]: This migration was indeed done on 22 march => at the time we thought we would keep textUrl
Yesterday, we decided it was not useful => so we delete

File: front/lib/resources/storage/models/content_fragment.ts (hunk 1 above)
Comment (philipperolet) [reply front/lib/resources/storage/models/content_fragment.ts:27]: we store sourceUrl

File: front/migrations/20240322_content_fragments_refactor.ts [hunk 3]
```diff
@@ -0,0 +1,150 @@
+import { Op } from "sequelize";
```
Comment (flvndvd) [inline front/migrations/20240322_content_fragments_refactor.ts:1]: Not to self, this migration was already run!

**Pull request merged**